ui/selectors/selectors.js

```diff
@@ -1226,7 +1200,7 @@ export const getMultipleTargetsSubjectMetadata = createDeepEqualSelector(
 
 export function getRpcPrefsForCurrentProvider(state) {
   const { rpcPrefs } = getProviderConfig(state);
-  return rpcPrefs || {};
```

Comment:
This is the only logic change in this PR. `getProviderConfig`'s type don't allow a falesy value for `rpcPrefs`, and elsewhere in our code requires the return value of this function to satisfy `RpcPrefsInterface` (so `{}` is invalid).